### DISCUSSION TITLE
Operator window popup on entering chat

### DISCUSSION THREAD

**Discussion by modF** [LiveHelperChat/livehelperchat#2012] [Q&A]
It has been some time since I have used LHC and I have recently upgraded and dusted things off again.  At some point I thought I saw  an option (or a module) to have a window popup for the operator upon accepting the chat.  I thought it was department based, but I am getting no where looking in the settings or the existing modules.   What would be the best way to accomplish this? 

To clarify, this is for the agent, not the visitor.

**Comment by remdex** [comment]
I'm not sure about what you are talking about here. In recent versions there is no such options enabled. Unless you go directly to chat last and open popup there.

**Reply by modF** [reply]
Sorry for the late reply, let me hopefully clarify a bit.  The chat would remain in the main window, but upon the operator on our end entering the chat a window would popup/out to an external system.  Something along time lines of something.com/search?email=email&phone=.   Could be used for a CRM, ticket system and things like that.

**Reply by remdex** [reply]
usually for that flow there is an extension which overrides one of the extension and there is a button to open external url. You can take a look ar osticket extension
